### PR TITLE
docs: shorten README and remove other docs

### PR DESCRIPTION
**Pull request by adamaltman**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,225 +13,24 @@ OpenAPI 3 CLI toolbox with rich validation and bundling features.
...
-    - "#/excluded/json/pointer"
-```
-
-Ignore file can be autogenerated to add all problems to it by running `openapi lint --generate-ignore-file`
-
-#### Built-in Rules
+## What makes this tool different
 
-[Read the docs](docs/rules.md) for the built-in rules.
+Unlike other OpenAPI linters, `@redocly/openapi-cli` defines the possible type tree of a valid OpenAPI definition and then traverses it. This approach is very similar to how compilers work and results in major performance benefits over other approaches. Extend functionality at different points in the lifecycle with preprocessors, rules, and decorators by following the [visitor pattern](https://en.wikipedia.org/wiki/Visitor_pattern).
```
Comment (RomanHotsiy) [inline README.md:30]:
```suggestion
Unlike other OpenAPI linters, `@redocly/openapi-cli` defines the possible type tree of a valid OpenAPI definition and then traverses it. This approach is very similar to how linters for programming languages work and results in major performance benefits over other approaches. Extend functionality at different points in the lifecycle with preprocessors, rules, and decorators.
```

File: README.md [hunk 2]
```diff
@@ -15,225 +13,24 @@ OpenAPI 3 CLI toolbox with rich validation and bundling features.
...
-- [x] Configurable message levels for each rule. You can tailor your experience with @redocly/openapi-cli as you wish.
-- [x] Lightning-fast validation. Check 1 Mb file in less than one second.
-- [x] Human-readable error messages. Now with stacktrace and codeframes.
+- [x] Lightning-fast validation. Lint a 1 Mb file in less than one second.
+- [x] Built-in rules for common validations.
+- [x] Configurable severity levels for each rule.
+- [x] Human-readable error messages with codeframes and stylish format options.
 - [x] Intuitive suggestions for misspelled types or references.
-- [x] Easy to implement custom rules. Need something? Ask us or do it yourself.
+- [x] Easy to implement custom rules.
```
Comment (RomanHotsiy) [inline README.md:21]: link to custom rules docs?

File: README.md [hunk 3]
```diff
@@ -15,225 +13,24 @@ OpenAPI 3 CLI toolbox with rich validation and bundling features.
...
+Unlike other OpenAPI linters, `@redocly/openapi-cli` defines the possible type tree of a valid OpenAPI definition and then traverses it. This approach is very similar to how compilers work and results in major performance benefits over other approaches. Extend functionality at different points in the lifecycle with preprocessors, rules, and decorators by following the [visitor pattern](https://en.wikipedia.org/wiki/Visitor_pattern).
 
-### Advanced
 
-[Plugins](docs/plugins.md)
-[Custom rules](docs/custom-rules.md)
-[Decorators](docs/decorators.md)
-[Preprocessors](docs/preprocessors.md)
-[Type extensions](docs/type-extensions.md)
+## [Read the docs](https://redoc.ly/docs/cli/)
```
Comment (RomanHotsiy) [inline README.md:33]: I would also add TLDR usage section, basically

`npx @redocly/openapi-cli lint path-to-file.yaml`

File: README.md (hunk 2 above)
Comment (adamaltman) [reply README.md:21]: There is a link to the docs below. When we have solid custom rules docs, we can add a deep link here to those.

Comment (RomanHotsiy) [reply README.md:21]: Good

**Pull request merged**